### PR TITLE
Add debugging to git clones.

### PR DESCRIPTION
**Pull request by feanil**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

File: playbooks/roles/git_clone/tasks/main.yml [hunk 1]
```diff
@@ -20,6 +20,14 @@
 # Rewrite this task using the ansible git-config module once we'll migrate to Ansible 2.x
 # https://docs.ansible.com/ansible/git_config_module.html#git-config
 
+- name: Debug Info about Repo
+  debug:
+    msg: "{{ item.PROTOCOL }} : {{ item.DOMAIN }} : {{ item.PATH }} : {{ item.REPO }} : {{ item.VERSION }} : {{ item.DESTINATION }}"
+  with_items: "{{ GIT_REPOS }}"
```
Comment (jibsheet) [inline playbooks/roles/git_clone/tasks/main.yml:26]: Are you sure ansible won't print out the entire contents of GIT_REPOS?  I thought that's what happened in the other place and why we turned on no_log (especially if someone is running ansible with -v)

Comment (feanil) [reply playbooks/roles/git_clone/tasks/main.yml:26]: I'll check in the travis runs.  You might be right.  I might not be able to get at this info.

Comment (jibsheet) [reply playbooks/roles/git_clone/tasks/main.yml:26]: If it does print, we can probably transform GIT_REPOS into another dictionary (or list of strings) using set_fact and having that step set no_log.  Then debug that list/dict.  Maybe.  I wonder if there's a reason I didn't already try that.

Comment (feanil) [reply playbooks/roles/git_clone/tasks/main.yml:26]: The deletion of a value from the original list is hard I think.  But yea, setting facts would be a good way to do it.

Comment (jibsheet) [reply playbooks/roles/git_clone/tasks/main.yml:26]: zip_to_dict consumes an args of of the vars we want, and an input of the dict, it might just work

**Pull request closed without merging**